src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/Internal/MsQuicAddressHelpers.cs

```diff
@@ -76,7 +59,7 @@ private static void SetPort(AddressFamily addressFamily, ref SOCKADDR_INET socke
                     break;
                 case AddressFamily.InterNetworkV6:
                 default:
-                    socketAddrInet.Ipv6._port = convertedPort;
+                    socketAddrInet.Ipv6.sin6_port = convertedPort;
```

Comment:
Nit: since the port field is at the same offset in both sockaddr structs, you don't actually need to switch on the addressFamily here.

Reply:
Done.